teek/_widgets/base.py

```diff
@@ -799,8 +801,9 @@ def __getitem__(self, sequence):
         for equiv_sequence, equiv_callback in self._callback_objects.items():
             # this equivalence check should handle corner cases imo because the
             # command names from create_command are unique
-            if (self._call_bind(str, sequence) ==
-                self._call_bind(str, equiv_sequence)):      # flake8: noqa
+            if (self._call_bind(str, sequence) == self._call_bind(
+                str, equiv_sequence
+            )):
```

Comment:
This is also less readable than the old code that was there, but I'll accept this if you really want to change the style for whatever reason.

Reply:
:+1: on this being less readable than the old line, at first glance it looked like an assignment to me